crates/nargo/Cargo.toml

```diff
@@ -26,9 +26,10 @@ tempdir = "0.3.7"
 
 # Backends
 aztec_backend = { optional = true, path = "../aztec_backend" }
+marlin_arkworks_backend = { optional = true, git = "https://github.com/noir-lang/marlin_arkworks_backend" }
 
 [features]
 default = ["plonk_bn254"]
 # The plonk backend can only use bn254, so we do not specify the field
 plonk_bn254 = ["aztec_backend"]
-# marlin_bn254 = ["acvm/marlin", "noir_field/bn254"]
+marlin = ["marlin_arkworks_backend"]
```

Comment:
So the curve is now specified inside the marlin repo?
How can I generate two proofs for 2 different marlin flavors, do I have to re-compile Noir to change the flavor?

Reply:
As discussed, it will require a compilation, but it was already like that before so this should be fixed but it is not urgent.